utilities/realsense-test/src/Module.cpp

```diff
@@ -0,0 +1,103 @@
+/**
+ * @file Module.cpp
+ * @authors Giulio Romualdi
```

Comment:
Personal opinion: it is quite easy to forget to update this metadata, and I am not really sure which benefit they provide (the name of the file is already given.. in the file name itself, and the authors are tracked with git). I would remove them, but probably we may want to do that consistently across the repo. cc @S-Dafarra

Reply:
> Personal opinion: it is quite easy to forget to update this metadata, and I am not really sure which benefit they provide (the name of the file is already given.. in the file name itself, and the authors are tracked with git). I would remove them, but probably we may want to do that consistently across the repo. cc @S-Dafarra

I agree. Maybe we can track this in an issue?

Reply:
Done in https://github.com/dic-iit/bipedal-locomotion-framework/pull/165/commits/e353d497152575cb9128c242cb8b823f788d3bff for files specific to this PR.